### ISSUE TITLE
Consistency in versions

### ISSUE THREAD

**Issue by magdalenafuentes** [mir-dataset-loaders/mirdata#339]
Before merging #338 I would like to discuss what to do with the versions every time we merge a PR in master. So far we're been a bit inconsistent, so should we agree on how to change the version depending if is a fix, new loader or breaking change? For instance, are we gonna treat them all as patches e.g. (X.Y.Z) increasing Z? Should we go for the `alpha`-`beta`-`rc` cycles from now on?

@rabitt @nkundiushuti thoughts?

**Comment by magdalenafuentes** [comment]
Whatever we decide should be written somewhere like CONTRIBUTING #290

**Comment by rabitt** [comment]
I think we don't need to change the version with every merge - so far I was trying to follow the convention (using your example) X.Y.Z, where:
* increase Z if we're releasing a minor patch
* increase Y if we're releasing a major change
* increase X for major API changes. So far we didn't increase X because the library is still in the early phase..

As for using beta - I think it's good for big changes that might break everything, like the Dataset class & the index changes. The reason for beta vs. changing Z is because of pypi - beta releases have to be explicitly installed (`pip install --pre mirdata`), where just doing `pip install mirdata` will install the latest non-beta release. We can move past beta when we feel everything is pretty stable!

All this to say, I think we the maintainers could manage the version updates, and we could instruct contributors to not change the version. We might want to let several PRs get merged before we upgrade Y/Z, depending on what the changes are.

@magdalenafuentes what do you think?

**Comment by nkundiushuti** [comment]
I agree we should increase X when there is a major change that breaks previous functionality (e.g. you can't write a new loader with the old API). 
We can group data loaders for a Y version. In addition we can increase Y with a major change. I can't think of an example right now.. 
And for minor changes and everything else we have a Z increase.
